docs/deploying-to-production.md

```diff
@@ -0,0 +1,121 @@
+# Deploying to production
+
+## Getting started
+
+Your production enviroment needs to have [Node.js](https://nodejs.org/) and
+[Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/) installed.
```

Comment:
Your production enviroment needs to have [Node.js](https://nodejs.org/) and [Yarn](https://yarnpkg.com/) installed.